### ISSUE TITLE
UIA in Word: braille is very redundant

### ISSUE THREAD

**Issue by dkager** [nvaccess/nvda#8407]
### Steps to reproduce:
1. Ensure that UIA is enabled for Word
2. Open Word

### Expected behavior:
- The braille display shows "Document1 - Word"

### Actual behavior:
- From the log: `Braille regions text: [u'Document1 - Word ', u'Document1 ', u'Document1 doc ', ' ']`
- That is "Document1 - Word Document1 Document1 doc"

### System configuration:
NVDA version:
master-15276,8d463d96

NVDA Installed or portable:
Installed

Other information:
The problem occurs all throughout Word:
- In the example document from #6233: "merged-cell-table.docx  -  Protected View - Word merged-cell-table.docx  -  Protected View pane merged-cell-table.docx [Read-Only] [Compatibility Mode] doc tbl r1 c1-2 tbl r1 c1-2 tbl r1 c1-2 Sea-dwelling"
- When you create a link: "lnk lnk lnk Click here"

Windows version:
10.0.17134

Name and version of other software in use when reproducing the issue:
Word 2016

**Comment by dkager** [comment]
CC @michaelDCurran

**Comment by michaelDCurran** [comment]
I could not reproduce the  link issue mentioned in this issue. It was most likely fixed in a later version of Word 2016.
Please ensure that MS Word is updated to the latest version. Either from the Ribon -> File -> Account, or in the Windows Store.

**Comment by michaelDCurran** [comment]
However, The first problem mentioned in this issue is now addressed by pr #8569.

**Comment by leonardder** [comment]
It sounds to me like there is something in the focus ancestry that should be treated as layout rather than content. This is also an issue with speech.
```
Input: kb(laptop):alt+tab
IO - speech.speak (20:48:32.415):
Speaking [u'Document1 - Word']
IO - speech.speak (20:48:32.515):
Speaking [u'Document1 - Word']
IO - speech.speak (20:48:32.555):
Speaking [u'Document1']
IO - speech.speak (20:48:32.558):
Speaking [u'Microsoft Word Document  edit  multi line']
IO - speech.speak (20:48:32.625):
Speaking [u'page 1  section 1', u'blank']
```
